### DISCUSSION TITLE
Tasmota Wifi not detected

### DISCUSSION THREAD

**Discussion by bkbartk** [arendst/Tasmota#16278] [Q&A] [answered]
Hello, I'm trying to flash a tasmota Dual R3 device.
and that went ok.
I actually did this 4 times, 2 times using the esp flasher,
and 2 times using the webinstaller.

unfortunately the tasmota-XXXX SID doesn't show up. Also not after unplugging ana inplugging the 3.3 volt.
According to the manual the webinstaller should suggest a wifi network to configure but it doesn't.

I tried reading the log but it's basically this in a loop.
```
0:00:00.003 HDW: ESP32-D0WD-V3 
00:00:00.042 UFS: FlashFS mounted with 308 kB free
00:00:00.085 CFG: Loaded from File, Count 2
00:00:00.100 QPC: Count 3
00:00:00.202 BRY: Berry initialized, RAM used=3735 bytes
00:00:00.229 Project tasmota - Tasmota Version 12.1.0.1(tasmota)-2_0_4_1(2022-08-21T12:50:46)
00:00:00.490 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DOUT, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:12784
load:0x40080400,len:3032
entry 0x400805e4
```
(the webinstaller has some info for BT beacon because I couldn't find plain Tasmota32)
but does this log make any sence to anyone?
Is my wifi antenna just broken or am I missing something here?

**Comment by sfromis** [comment, marked as answer]
Some devices cannot get on Wifi with 3.3V power only, but need to be connected to mains power instead.

It is perfectly normal for Tasmota to reboot every 3 minutes while yearning for wifi. In the web installer, do not look for special ESP32 builds, when you just select the default Tasmota, the web installer will detect the chip, and choose the relevant variant.

**Reply by bkbartk** [reply]
plugging it in to a wall socket enabled WiFi. 

and for the next time I know which version to select. Now, after being able to enter the webinterface I just used firmware update to get to the default version.
